README.md

```diff
@@ -109,3 +109,17 @@ docker run -d -p 3000:3000 --name=grafana -v /tmp/grafana-data:/var/lib/grafana
 Please have a look at the documentation of each image/application for adequate configuration examples.
 
 A simple but complete example to deploy a full monitoring stack on kubernetes can be found in the [examples](examples/config/k8s) directory
+
+## Monitoring On-Premise with Prometheus and Grafana
+Take holistic overview of your Pure Storage FlashArray estate on-preimise with Prometheus and Grafana to summerise statistics such as:
```

Comment:
Take a holistic...

Reply:
Thanks for catching these @sdodsley - corrections made in commit#3